model/src/main/java/org/cbioportal/model/Gene.java

```diff
@@ -49,4 +51,26 @@ public Integer getLength() {
     public void setLength(Integer length) {
         this.length = length;
     }
+    
+    public String getChromosome() {
```

Comment:
Almost all of the new Model classes have no functionality aside from getters, setters, and maybe constructors.
Since we area using the model classes to serve the needs across all layers (persistence, service, web) I think we need to be careful about building in functionality which is not appropriate for all layers. But I think that these simple utility functions are not layer-specific, are generally helpful, and it would be worse to create some kind of "GeneChromosomeParser" class. One question though: some genes are located on both the X and Y chromosomes (see below). Should we distinguish between genes only on X, genes only on Y, and genes on both X and Y ?
mysql> select * from gene where cytoband like "%;%" and TYPE="protein-coding";
+----------------+------------------+----------------+------------------+--------+
| ENTREZ_GENE_ID | HUGO_GENE_SYMBOL | TYPE           | CYTOBAND         | LENGTH |
+----------------+------------------+----------------+------------------+--------+
|         286530 | P2RY8            | protein-coding | Xp22.33; Yp11.3  |   4459 |
|           6473 | SHOX             | protein-coding | Xp22.33;Yp11.3   |   6114 |
|           9189 | ZBED1            | protein-coding | Xp22.33;Yp11     |   4755 |
|           8225 | GTPBP6           | protein-coding | Xp22.33; Yp11.32 |   2363 |
|           8623 | ASMTL            | protein-coding | Xp22.3; Yp11.3   |   3506 |
|          28227 | PPP2R3B          | protein-coding | Xp22.33; Yp11.3  |   9015 |
|          55344 | PLCXD1           | protein-coding | Xp22.33; Yp11.32 |   6191 |
|          64109 | CRLF2            | protein-coding | Xp22.3; Yp11.3   |   1205 |
|         207063 | DHRSX            | protein-coding | Xp22.33; Yp11.2  |   3620 |
+----------------+------------------+----------------+------------------+--------+